functions/notify_slack.py

```diff
@@ -41,6 +41,45 @@ def cloudwatch_notification(message, region):
   }
 
 
+def guardduty_finding(message):
+  states = {'Low': '#777777', 'Medium': 'warning', 'High': 'danger'}
+  region = message['region']
```

Comment:
any reason you didn't pass the region into the function similar to the cloudwatch_notification one